Source/DafnyDriver/Commands/ProofComplexityCommand.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Linq;
+
+namespace Microsoft.Dafny;
+
+public class ProofComplexityCommand : ICommandSpec {
+  public IEnumerable<Option> Options => new Option[] {
+    Iterations,
+    RandomSeed,
+    Format,
+    IsolateAssertions,
+  }.Concat(ICommandSpec.VerificationOptions).
+    Concat(ICommandSpec.CommonOptions);
+
+  static ProofComplexityCommand() {
+    DafnyOptions.RegisterLegacyBinding(Iterations, (o, v) => o.RandomSeedIterations = (int)v);
+    DafnyOptions.RegisterLegacyBinding(RandomSeed, (o, v) => o.RandomSeed = (int)v);
+    DafnyOptions.RegisterLegacyBinding(IsolateAssertions, (o, v) => o.VcsSplitOnEveryAssert = v);
+    DafnyOptions.RegisterLegacyBinding(Format, (o, v) => o.VerificationLoggerConfigs = v);
+  }
+  private static readonly Option<uint> Iterations = new("iterations", () => 10U,
+    $"Attempt to verify each proof n times with n random seeds, each seed derived from the previous one. {RandomSeed.Name} can be used to specify the first seed, which will otherwise be 0.") {
+    ArgumentHelpName = "n"
+  };
+
+  private static readonly Option<uint> RandomSeed = new("random-seed", () => 0U,
```

Comment:
You mean `RandomSeed` and `IsolateAssertions` ? What's the use-case?

Do you think a Dafny project could be developed with `--isolate-assertions` on at all times? If not, how would you use it?

If the durability/complexity checker complains that a proof is too complex (saying a proof is not durable doesn't sound right), then you could think a user might want to do `dafny verify --random-seed=<problematic-seed>` to improve the proof. However, in that case I'd suggest they use `dafny check-proof-durability --random-seed=<problematic-seed>` instead, because then they'll be able to see whether they introduce other problematic seeds.

Reply:
> Do you think a Dafny project could be developed with --isolate-assertions on at all times? If not, how would you use it?

It certainly could be, even though it's unlikely to scale well enough for typical projects. But I could even imagine degenerate cases of a small project with just a small number of really large methods where that would work well.

> (saying a proof is not durable doesn't sound right)

Agreed and see my top-level comment.

> then you could think a user might want to do dafny verify --random-seed=<problematic-seed> to improve the proof

Yeah, I don't think we'd want to encourage that except as a short-term hack, since it's likely not addressing the root cause of proofs that are too complex. But it would be useful in those cases, especially if you really just want to know if your code is valid and aren't yet concerned with durability.